INSTALL

```diff
@@ -0,0 +1,169 @@
+           Netatalk Installation
+           =====================
+
+Quick start
+===========
+
+Netatalk can be built by using Meson or GNU Autotools.
+The meson build system is the recommended build system for the master
```

Comment:
Terminology nitpick, `main` branch instead of `master`

Reply:
Fixed